ocs_ci/ocs/resources/objectconfigfile.py

```diff
@@ -135,3 +136,57 @@ def get(self, namespace=None):
         """
         out = self._run_command("get", namespace, out_yaml_format=True)
         return yaml.safe_load(out)
+
+    def describe(self, namespace=None):
+        """
+        Run ``oc describe`` on in this object file.
+
+        Args:
+            namespace (str): Name of the namespace where to deploy, overriding
+            self.project.namespace value (in a similar way how you can specify
+            any value to ``-n`` option of ``oc describe``.
+        """
+        out = self._run_command("describe", namespace, out_yaml_format=False)
+        return yaml.safe_load(out)
+
+    def wait_for_delete(self, resource_name="", timeout=60, sleep=3, namespace=None):
+        """
+        Wait for a resource to be deleted
+
```

Comment:
What will happen in case you provide a resource name from a different namespace? 
The resource will not be deleted but you will get a message that it indeed was, right ? 

Shouldn't we add a check for verifying that the resource belongs to the namespace ?

Reply:
@ypersky1980  Adding resource name alone is not going to help at any case, since that cmd is going to be executed in the particular default or the namespace set at that time, and might fail/pass.
**Please note** prior to this function there is a call to delete the kube_job obj, and this function is more of waiting for that resource to be deleted, which is similar to other wait_for_resource function in ocs-ci, If the namespace is not matching then obvious it's going to fail.
We don't need this verification separately, it's going to add more overhead and we do know which namespace we are using for clone creations. let me know WDYT?

Reply:
Thank you for the explanation, agree.